cypress/integration/test_table.js

```diff
@@ -0,0 +1,81 @@
+/// <reference types="Cypress" />
+
+describe("tests table", function() {
+  this.beforeEach(() => {
+    cy.server();
+    cy.login();
```

Comment:
this makes it so that cypress logs in before each of the `it` statements below. looks to me like logging in once before all tests using `beforeAll` would suffice.

Reply:
tried writing some tests myself and it looks like because of how auth currently works that logging in before every approach is required for now. my first ticket this sprint is to automatically authenticate a user. once that is done we should only need to authenticate once per test suite.

Reply:
@tgrander do you still request changes here?